backend/src/plugins/authorization/authorization.plugin.ts

```diff
@@ -11,6 +15,12 @@ type Options = {
   };
 };
 
+declare module 'fastify' {
+  interface FastifyRequest {
+    user: EAMMasterSignInResponseDto | EAMWorkerSignInResponseDto;
+  }
+}
```

Comment:
```suggestion
declare module 'fastify' {
  interface FastifyRequest {
    user?: EAMMasterSignInResponseDto | EAMWorkerSignInResponseDto;
  }
}
```
move to `src/index.d.ts`